src/installation/live-images/prep.md

```diff
@@ -1,27 +1,43 @@
-# Preparing installation media
+# Prepare Your Installation Media
 
-After [obtaining an image](./downloading.md), it must be written to a storage
-device, such as a USB drive, SD card, or CD/DVD.
+After [downloading a live image](./downloading.md), it must be written to
+bootable media, such as a USB drive, SD card, or CD/DVD.
 
-## Create a bootable USB drive or SD card on Linux
+## Create a Bootable USB Drive or SD Card on Linux
 
-The `dd` command can be used to copy a live image to a storage device.
+### Identify Your Device
 
+Before we write the image, we need to identify the device we want to write it
```

Comment:
Please rephrase to be of the form "Before writing the image..."

Reply:
Rephrased as such. Not sure if it sounds too terse after fitting the sentence to that. Could rewrite it down to just the following if it reads better:

> Before writing the image, identify the device you'll write it to using [fdisk(8)](https://man.voidlinux.org/man8/fdisk.8). After connecting the storage device, identify the device path by running: